frontend/justfile

```
@@ -71,3 +71,19 @@ run *args:
 
 types:
     cd .. && pnpm exec vue-tsc -p frontend --noEmit
+
+##########
+# Health #
+##########
+
+# Check the health of the service
+@health host="localhost:50288":
+    -curl -s -o /dev/null -w '%{http_code}' 'http://{{ host }}/api/health'
+
+# Wait for the service to be healthy
+@wait host="localhost:50288":
+    # The just command on the second line is executed in the context of the
+    # parent directory and so must be prefixed with `api/`.
+    just ../_loop \
+    '"$(just frontend/health {{ host }})" != "200"' \
+    "Waiting for the frontend to be healthy..."
```

Along the same lines as @sarayourfriend's feedback, this should be updated as follows.

```suggestion
    "Waiting for Plausible to be healthy..."
```